### PR TITLE
Create rainwatertrapping.py

### PR DESCRIPTION
**Pull request by YashGupta1510**
### Describe your change:
Added a solution to the Rain Water Trapping problem using the two pointer approach as I couldn't find this solution or any other algorithm with the two pointer approach

### CODE REVIEW

File: other/rainwatertrapping.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            if arr[i] >= prev:
+                prev = arr[i]
+            else:
+                water += prev - arr[i]
+    return water
+
+
+if __name__ == "__main__":
+    arr = [0, 6, 0, 4, 1, 3, 2, 6, 1, 2, 1, 3]
+    N = len(arr)
```
Comment (CaedenPH) [inline other/rainwatertrapping.py:29]:
```suggestion
```

File: other/rainwatertrapping.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+def maxTrappedWater(arr, n):
```
Comment (CaedenPH) [inline other/rainwatertrapping.py:1]:
```suggestion
def maxTrappedWater(arr):
    n = len(arr)
```
You should get the length inside, not pass it as a parameter

[the remark this answers is not in the document]
Comment (YashGupta1510) [reply other/rainwatertrapping.py:1]: integer

**Comment by cclauss** [comment]
Please read CONTRIBUTING.md before submitting another pull request.

**Pull request closed without merging**